hooks/lib/pdlc-test-strategy.sh

```
@@ -0,0 +1,145 @@
+# hooks/lib/pdlc-test-strategy.sh — Test strategy recommendation engine
+#
+# Analyzes spec.md and plan.md to recommend test types, frameworks,
+# coverage targets, and TDD guidance for Actor sessions.
+# Read-only Observer: reports recommendations, never blocks.
+#
+# Sourced by other scripts: source "$(dirname "$0")/lib/pdlc-test-strategy.sh"
+# Depends on: (none — standalone analysis)
+
+set -euo pipefail
+
+TEST_STRATEGY_LIB_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] TEST_STRATEGY_LIB_DIR appears unused. Verify use (or export if used externally).

(SC2034)